src/main.jsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
+import '@fontsource/julius-sans-one';
+import '@fontsource/alice';
 import './index.css';
+import { ChakraProvider } from '@chakra-ui/react';
+import theme from './theme.jsx';
```

Comment:
Il faut quand même qu'il soit dans un dossier, raison d'orga 👀 
```suggestion
import theme from './theme/theme.jsx';
```